students/synowiec_krzysztof/lesson_06_dicts_tuples_sets_args_kwargs/access_rights.py

```diff
@@ -0,0 +1,22 @@
+def main():
+    operations_dict = {
+        'read': 'R',
+        'write': 'W',
+        'execute': 'X',
+    }
+
+    file_permissions = {}
+    for x in range(int(input())):
+        file, *operations = input().split()
+        file_permissions[file] = set(operations)
```

Comment:
I know that it would work with: "file_permissions[file] = operations" but it seemed to me that there is duplicated permission in one of snakify test. So I decided to use set. I can not find this example know (probably it was never there ;) ) anyway it makes code resistant from permission duplicates.

Reply:
good and explanation. me like it ;)